### PR TITLE
Joomla! 3.7 Custom admin menu support.

### PR DESCRIPTION
**Pull request by tonypartridge**

### CODE REVIEW

File: component/site/views/default/helpers/defaultloadedfromtemplate.php [hunk 1]
```diff
@@ -881,7 +890,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 
 								$row = $event;
 								$times = "";
-								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat" || $jevtask == "range")
+								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat")
```
Comment (GeraintEdwards) [inline component/site/views/default/helpers/defaultloadedfromtemplate.php]: Perhaps you should close the other pull request and stick to this one.

what is the purpose of this change?

File: component/site/views/default/helpers/defaultloadedfromtemplate.php [hunk 2]
```diff
@@ -925,7 +934,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 										$times = $start_date . ' - ' . $stop_date . " " . $times . "<br/>";
 									}
 								}
-								else if (($jevtask == "day" || $jevtask == "week" ) && ($row->starttime() != $row->endtime()) && !($row->alldayevent()))
+								else if (($jevtask == "day" || $jevtask == "week"  || $jevtask == "range") && ($row->starttime() != $row->endtime()) && !($row->alldayevent()))
```
Comment (GeraintEdwards) [inline component/site/views/default/helpers/defaultloadedfromtemplate.php]: as above

Comment (tonypartridge) [inline component/site/views/default/helpers/defaultloadedfromtemplate.php]: See my other pull request. I've commented on :-)

File: component/site/views/default/helpers/defaultloadedfromtemplate.php (hunk 1 above)
Comment (tonypartridge) [inline component/site/views/default/helpers/defaultloadedfromtemplate.php]: Good point, I was a dufus and created the 3.7 repo from dev2 repo :-(. From the other pull:

I've merged so many times it's lost the original pull naming. 

Reasoning behind this is uniformity as requested b @brianteeman Full commit to this one here: https://github.com/JEvents/JEvents/pull/361/commits/407d7ab1639bc1852d383295126f5f5bc11243c5

Range views now always have a date column to the left. So tend not to need the date output again. Any users requiring the date and do so easily with the additional date string.

Comment (tonypartridge) [inline component/site/views/default/helpers/defaultloadedfromtemplate.php]: Also see:
https://www.jevents.net/discussions/empty-custom-fields

@GeraintEdwards
